lib/amazon-pricing/definitions/ec2-instance-type.rb

```diff
@@ -114,5 +132,130 @@ def self.get_name(instance_type, api_name, is_reserved = false)
       [api_name, name]
     end
 
+    # throughput values for performance ratings
+    # Amazon informally tells customers that they translate as follows:
+    #   Low = 100 Mbps
+    #   Medium = 500 Mbps
+    #   High = 1 Gbps
+    #   10Gig = 10 Gbps (obviously)
+    @Network_Throughput_Bytes_Per_Hour = {
+        :very_low => 50,    # FIXME check this value
```

Comment:
what needs to be done here? how do we get throughput number for :very_low today?